mitodl/odl-video-service
# refactoring CollectionDetail, in preparation for videos pagination

#### What are the relevant tickets?
Preparation for front-end part of #503 .

#### What's this PR do?
Refactors `CollectionDetailPage`:

1. Breaks out videos list into its own component (this is where pagination will happen)
2. Refactors `CollectionDetailPage_test.js` into unit tests using shallow
3. Harmonizes property names between video components.

#### How should this be manually tested?
Click around the collection detail page, ensure that everything looks as it did before.
